doc/configuration.md

```diff
@@ -2,7 +2,48 @@
 
 The Ignition configuration is a JSON document conforming to the following specification, with **_italicized_** entries being optional:
 
-* **ignitionVersion** (integer): the version number of the spec. Must be `1`.
+* **ignition** (object): metadata about the configuration itself.
+  * **version** (string): the semantic version number of the spec. Must be at least `2.0.0`.
+  * **_append_** (object): append one or more configs to the current config.
+    * **_configs_** (list of objects): a list of the configs to be appended.
+      * **source** (object): the source of the config.
+        * **uri** (string): the URI of the remote config.
+      * **_verification_** (object): options related to the verification of the config.
```

Comment:
On the one hand, I appreciate the flexibility/future proofing; on the other hand, the nesting is awfully hairy. I wonder if we can be more opinionated, with more assumptions (e.g., only support detached signatures), and consequently be able to flatten and simplify this schema a bit.